Author: Ondřej Nový <onovy@debian.org>
Description: Remove Carbod Ads
Origin: https://github.com/pallets/flask/commit/ba868d681aac8273ad785ebf28f4a043b737ab78

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -47,20 +47,17 @@
             'Issue Tracker', 'https://github.com/pallets/flask/issues/'),
     ],
     'canonical_url': 'http://flask.pocoo.org/docs/{}/'.format(version),
-    'carbon_ads_args': 'zoneid=1673&serve=C6AILKT&placement=pocooorg',
 }
 html_sidebars = {
     'index': [
         'project.html',
         'versions.html',
-        'carbon_ads.html',
         'searchbox.html',
     ],
     '**': [
         'localtoc.html',
         'relations.html',
         'versions.html',
-        'carbon_ads.html',
         'searchbox.html',
     ]
 }
